front/src/modules/trainschedule/components/ManageTrainSchedule/PacedTrainSettings/AddedOccurences.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from 'react';
+
+import { Clock, Calendar, Trash, Dot } from '@osrd-project/ui-icons';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+
+import InputSNCF from 'common/BootstrapSNCF/InputSNCF';
+import { addException, deleteException } from 'reducers/osrdconf/operationalStudiesConf';
+import { getExceptions } from 'reducers/osrdconf/operationalStudiesConf/selectors';
+import { useAppDispatch } from 'store';
+
+const AddedOccurences = () => {
+  const [date, setDate] = useState('');
+  const [time, setTime] = useState('');
```

Comment:
In my opinion, it would be simpler to have a single state holding a `Date`.

Reply:
Hm, I suppose it would be a bit cumbersome in event handlers though… Feel free to ignore.

Reply:
yeah, That’s what I was doing but I reverted to this version. much less code to just merge into one Date when we create the exception.